res/css/views/messages/_MLocationBody.scss

```diff
@@ -23,6 +23,9 @@ limitations under the License.
         border-radius: $timeline-image-border-radius;
         cursor: pointer;
     }
+    .mx_MLocationBody_tooltip{
```

Comment:
Missing space before `{`

Reply:
Actually, I was trying using important, otherwise, it would take z-index 0. But still, the tooltip is overlapping.
I am not able to figure out the reason.